flex-squid/src/test/java/org/sonar/flex/grammar/directives/AttributesTest.java

```diff
@@ -31,7 +31,9 @@
   @Test
   public void test() {
     Assertions.assertThat(g.rule(FlexGrammar.ATTRIBUTES))
```

Comment:
@lindamartin The [reported](http://jira.sonarsource.com/browse/SONARFLEX-41) parse error was on ```public static native function get ptype():String;``` and my understanding is that native was not considered as a valid Attribute. Am I missing something?
I updated IdentifierTest in 1d520a9.